nncf/openvino/optimized_functions/models.py

```diff
@@ -115,6 +117,16 @@ def clear_ov_model_cache():
     OV_MODEL_CACHE.clear()
 
 
+def _compile_ov_model(model: ov.Model, device_name: str, config: Dict[str, str]) -> ov.CompiledModel:
+    if is_lnl_cpu():
+        with set_env_variable("DNNL_MAX_CPU_ISA", "AVX2_VNNI"):
```

Comment:
I believe that the fix should be pushed to the develop branch as well. Keep in mind that this fix is only applicable for openvino <= 2025.0

Reply:
Added the check